gateway/1.6/modules/ROOT/pages/policies-included-schema-validation.adoc

```diff
@@ -17,7 +17,7 @@ endif::[]
 
 == Summary
 
-NOTE: The Schema Validation Policy currently only supports OpenAPI Specification 3.0 (OAS3) schema validations.
+NOTE: The Schema Validation Policy only supports REST APIs and currently only supports OpenAPI Specification 3.0 (OAS3) schema validations.
```

Comment:
Should policy be lowercase, or does it appear init cap in the UI? Line 20 of policies-included-json-threat-protection.adoc has policy lowercase.